src/middleware/built-in.ts

```
@@ -0,0 +1,393 @@
+  return (
+    req: NextRushRequest,
+    res: NextRushResponse,
+    next: () => void
+  ): void => {
+    const acceptEncoding = req.headers['accept-encoding'] || '';
+
+    if (
+      !acceptEncoding.includes('gzip') &&
+      !acceptEncoding.includes('deflate')
+    ) {
+      return next();
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)